bin/bazel_to_go.py

```diff
@@ -127,20 +132,26 @@ def bazel_to_vendor(WKSPC):
         print "WORKSPACE file not found in " + WKSPC
         print "prog BAZEL_WORKSPACE_DIR"
         return -1
+    mixer_adapter_repos = WKSPC + "/adapter_author_deps.bzl"
```

Comment:
No necessary in this PR, but this kind of parsing can be done by `bazel query`

Feed the output of `bazel query 'kind(go_repository, //external:*)' --output=build` to the AST here would be better :)